tests/test_api.py

```diff
@@ -106,10 +110,9 @@ def test_connection_ssl_error_verbose(self):
             with settings.runtime_values(verbose=True, host='https://foo.co'):
                 with mock.patch.object(Session, 'request') as req:
                     req.side_effect = ErrorType
-                    with mock.patch.object(debug, 'log') as dlog:
+                    with mock.patch.object(debug, 'log') as dlog:  # noqa
```

Comment:
As best as I can tell, this patch is really only here to keep the tests from spewing out a bunch of SSL warnings in the test output.